packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Allows for profile specification on `zowe config secure` command to avoid prompting for other profiles
```

Comment:
Can we include a related PR/issue link to this? 
Confused about the part that says "other profiles." The command is updated to allow the user to specify a profile to avoid the system from asking for _other_ profile info?